service/src/main/kotlin/fi/espoo/evaka/application/PendingDecisionEmailService.kt

```diff
@@ -97,6 +99,15 @@ GROUP BY application.guardian_id
         return jobCount
     }
 
+    fun getFromAddress(lang: Language, env: Environment): String {
+        val fromAddress = env.getRequiredProperty("mail_reply_to_address")
+        val foo = env.getRequiredProperty("mail_sender_name")
+        return when (lang) {
+            Language.sv -> "$foo <$fromAddress>"
+            else -> "$foo <$fromAddress>"
+        }
```

Comment:
But that's the same? Any idea if these could/should be used here (found in evaka-infra env): `APPLICATION_EMAIL_NAME_FI` & `APPLICATION_EMAIL_NAME_SV`?